js/models/Profile.js

```diff
@@ -118,7 +94,8 @@ export default class extends BaseModel {
       // dupes after the first one
       if (socialAttrs.type !== 'other' && groupedByType[socialAttrs.type].length > 1 &&
         groupedByType[socialAttrs.type].indexOf(socialMd) > 0) {
-        addError(`social[${index}].type`, 'You already have a social account of this type.');
```

Comment:
This one I do need to translate. I thought it wouldn't make it into the UI, but it very well can.